docker-for-windows/troubleshoot.md

```diff
@@ -316,6 +325,12 @@ Also, see the user-reported issue [Unable to run Docker for Windows on Windows
 10 Enterprise](https://github.com/docker/for-win/issues/74).
 
 
+### Virtualization on Windows Bootcamp (Mac)
```

Comment:
This section basically explains how to use macOS Bootcamp. I think we should not include this in our docs, since they should focus on what's specific to Docker, and this information can be found anywhere else.

It might seem that you're helping users but down the line adding non-critical information to this doc will make it harder for users to troubleshoot when they really have problems.

Reply:
I was solving my own use-case problem here & figured it'd be useful for others-- definitely see your point, this doc is already a bit unwieldy as is. I'll remove and stuck folks can search around for this slightly esoteric Windows-on-Mac-running-bootcamp scenario =)

Filed an issue on bootcamp+docker-for-windows specifically to get the issue off of this pr: https://github.com/docker/docker.github.io/issues/6465